lib/ansible/modules/crypto/get_certificate.py

```diff
@@ -44,8 +44,7 @@
   - When using ca_cert on OS X it has been reported that in some conditions the validate will always succeed.
 
 requirements:
-  - "python >= 2.6"
-  - "python-pyOpenSSL >= 0.15"
+  - "pyOpenSSL >= 0.15"
```

Comment:
Personally I'd much prefer if al these dependencies were URLs linking to PyPi or some project homepage, but that would be ideally an Ansible-wide change.

Reply:
Yes, that would indeed be nice. Do you want to create a feature request issue for that? (If you do, please CC me.)